partcad/src/partcad/ai_feature_file.py

```
@@ -0,0 +1,92 @@
+# Licensed under Apache License, Version 2.0.
+#
+
+import tempfile
+import re
+import requests
+
+from . import logging as pc_logging
+
+
+class AiContentFile:
+    def __init__(self, url: str):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)